transformations/stopword_removal/transformation.py

```diff
@@ -0,0 +1,40 @@
+from nltk.corpus import stopwords
+from nltk.tokenize import ToktokTokenizer
+from nltk.tokenize.treebank import TreebankWordDetokenizer
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def stopword_remove(text, max_outputs=1):
+    """
```

Comment:
the `max_outputs` argument isn't used anywhere in the `stopword_remove` function?

Reply:
I left it there as I noticed it to be a part of all the other transformations in the library. Will leave it there unless explicitly told otherwise!